lib/pbench/test/unit/server/test_server_configuration.py

```diff
@@ -154,10 +156,33 @@ def test_put_redundant_value(self, query_put):
     def test_put_param(self, query_put):
         response = query_put(key="dataset-lifetime", query_string={"value": "2"})
         assert response.json == {"dataset-lifetime": "2"}
+        audit = Audit.query()
+        assert len(audit) == 2
+        assert audit[0].operation == OperationCode.UPDATE
+        assert audit[0].status == AuditStatus.BEGIN
+        assert audit[0].name == "config"
+        assert audit[0].object_type == AuditType.CONFIG
+        assert audit[0].object_name is None
+        assert audit[1].operation == OperationCode.UPDATE
+        assert audit[1].status == AuditStatus.SUCCESS
+        assert audit[1].name == "config"
+        assert audit[1].object_type == AuditType.CONFIG
+        assert audit[1].object_name is None
+        assert audit[1].attributes["updated"] == {"dataset-lifetime": "2"}
 
     def test_put_value(self, query_put):
         response = query_put(key="dataset-lifetime", json={"value": "2"})
         assert response.json == {"dataset-lifetime": "2"}
+        audit = Audit.query()
+        assert len(audit) == 2
+        assert audit[0].operation == OperationCode.UPDATE
+        assert audit[0].status == AuditStatus.BEGIN
+        assert audit[0].name == "config"
+        assert audit[0].object_type == AuditType.CONFIG
+        assert audit[1].operation == OperationCode.UPDATE
+        assert audit[1].status == AuditStatus.SUCCESS
+        assert audit[1].name == "config"
+        assert audit[1].attributes["updated"] == {"dataset-lifetime": "2"}
```

Comment:
Why do we test only four fields from each record (i.e., we test five or six in the previous and subsequent tests), and why aren't we testing the _same_ four fields here?

Likewise, line 239?

Reply:
The only one missing here is `object_name`, which should always be `None` for config records and was verified as such earlier.

I guess I didn't check `[1].object_type`, though it's a constant for config operations and I have trouble getting very excited about that omission (given it's already be verified). Sure, I'll add it.